src/services/events/leafletDrawEvents.js

```diff
@@ -0,0 +1,26 @@
+angular.module('ui-leaflet')
+.factory('leafletDrawEvents', function(leafletEventsHelpersFactory) {
+
+  class DrawEvents extends leafletEventsHelpersFactory{
```

Comment:
BTW I thought the Babel compiled code to this was quite comical. As people who hate CoffeeScript babel's output is way way harder to read. I almost would rather revert this to ES5.